src/main/kotlin/com/hibob/academy/dao/tables.kt

```diff
@@ -0,0 +1,26 @@
+package com.hibob.academy.dao
+
+import com.hibob.academy.utils.JooqTable
+
+class PetTable(tableName: String = "pet"): JooqTable(tableName) {
+
```

Comment:
What about id field?
Do we want to query by id as well? If so then we need to add it as well here.